src/relentless/simulate/hoomd.py

```diff
@@ -212,7 +212,30 @@ def _convert_to_gsd_file(self, sim):
                 gsd_filename = None
             gsd_filename = mpi.world.bcast(gsd_filename)
         else:
-            gsd_filename = self.filename
+            if mpi.world.rank_is_root:
+                gsd_filename = self.filename
+                with gsd.hoomd.open(self.filename) as snap:
+                    frame = snap[0]
+                    if frame.configuration.dimensions == 2:
+                        if _hoomd_version.major >= 3:
+                            if frame.configuration.box[2] == 0.0:
+                                pass
+                            else:
+                                frame.configuration.box[2] = 0
+                                gsd_filename = sim.directory.temporary_file(".gsd")
+                                with gsd.hoomd.open(gsd_filename, _gsd_write_mode) as f:
+                                    f.append(frame)
+                        else:
+                            if frame.configuration.box[2] > 0.0:
+                                pass
+                            else:
+                                frame.configuration.box[2] = 1
+                                gsd_filename = sim.directory.temporary_file(".gsd")
+                                with gsd.hoomd.open(gsd_filename, _gsd_write_mode) as f:
+                                    f.append(frame)
```

Comment:
Consolidate these into an `if-elif-else` (no empty if & pass). You can add comments about why these particular conditions are needed:
```
if _hoomd_version.major >= 3 and frame.configuration.box[2] == 0:
    # fix for HOOMD 2 style used in HOOMD 3
elif _hoomd_version.major == 2 and frame.configuration.box[2] > 0:
    # fix for HOOMD 3 style used in HOOMD 2
else:
    gsd_filename = self.filename
```